modular_sand/code/modules/mob/dead/new_player/sprite_accessories/tails.dm

```diff
@@ -30,3 +30,15 @@
 	icon = 'modular_sand/icons/mob/mam_tails.dmi'
 	color_src = MATRIXED
 	matrixed_sections = MATRIX_RED_GREEN
+
+/datum/sprite_accessory/tails/mam_tails/blade //Ported from Vorestation
+	name = "Blade"
+	icon_state = "blade"
+	icon = 'modular_sand/icons/mob/mam_tails.dmi'
+	color_src = MUTCOLORS
+
+/datum/sprite_accessory/tails_animated/mam_tails_animated/blade //Ported from Vorestation
+	name = "Blade"
+	icon_state = "blade"
+	icon = 'modular_sand/icons/mob/mam_tails.dmi'
+	color_src = MUTCOLORS
```

Comment:
I think this is how white matrixed files are handled but IDK

Reply:
Did you test it in-game?

Reply:
Yes

Sprite is just darker than most, which I think it due to how it’s matrixed